### ISSUE TITLE
How to get the game to crash. (Somewhat inconsistently.)

### ISSUE THREAD

**Issue by cursedfoxy** [bunfan/beat-banger-public#231]
### Discord Tag

@cursedfoxy

### Issue Description

I was messing around with the game.

I went to extras
>Music
>Did the Konami Code to unlock the last two music disks
>Pick either of the two disks.
>go back to main menu
>Now there's seems to be a chance of the game crashing whenever you go into any of the game menus from this point forwards.

it consistently crashes for me whenever I hit SHOP.

https://github.com/bunfan/beat-banger-public/assets/47169904/360f825d-36fd-4990-a259-fd45b6b91d3e

### OS

Windows

### Version

3.34 dev1 (Steam)

### Relevant log output

```shell
Godot Engine v4.2.stable.official.46dc27791 - https://godotengine.org
OpenGL API 3.3.0 NVIDIA 551.23 - Compatibility - Using Device: NVIDIA - NVIDIA GeForce GTX 980
 
13:48:04: INFO Loading Save Data...
13:48:04: INFO Validating Save...
13:48:04: INFO Saving Game...
13:48:04: INFO Loading User Settings...
USER ERROR: Unable to convert array index 0 from 'Nil' to 'String'.
   at: assign (core/variant/array.cpp:245)
13:48:04: INFO Validating Keybinds...
13:48:04: INFO Loading Keybinds...
WARNING: AnimationMixer: 'RESET', Value Track: '.:default_color' has different update modes between some animations which may be blended together. Blending prioritizes UpdateMode.UPDATE_CONTINUOUS, so the process treats UpdateMode.UPDATE_DISCRETE as UpdateMode.UPDATE_CONTINUOUS with InterpolationType.INTERPOLATION_NEAREST.
   at: _update_caches (scene/animation/animation_mixer.cpp:889)
```

**Comment by cursedfoxy** [comment]
https://github.com/bunfan/beat-banger-public/assets/47169904/25bf39a0-2b37-40eb-a8bd-64a0031fd32b

The same thing with the second locked music disk.

**Comment by cursedfoxy** [comment]
I guess changing the main menu theme in music causes the issue?

**Comment by Stefan-Mitric** [comment]
closed because duplicate of #204 and already fix in Version 3.34 as an small patch.
